src/main/java/seedu/address/model/assignment/Status.java

```diff
@@ -2,12 +2,20 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class Status {
+public class Status implements Comparable<Status> {
     public static final String MESSAGE_CONSTRAINTS = "Status of assignment should be clearly defined!";
     public final String value;
 
     enum StatusType {
-        COMPLETED, LATE, PENDING
+        COMPLETED(2),
```

Comment:
I'm not too sure about enumerations but this COMPLETED(2) is like class instantiation with a weight field of value 2? Does it affect the string representation of the COMPLETED status?